lib/page-object/page_populator.rb

```diff
@@ -58,9 +58,9 @@ def populate_radiobuttongroup(key, value)
       return self.send("select_#{key}", value)
     end
 
-  def populate_select_list(key, value)
+    def populate_select_list(key, value)
```

Comment:
Doesn't this code mean that `#page_populate_with` behaves differently than when using the setter defined by the accessors? I would think that this conditional logic for text vs value would exist the accessor method (or eventually Watir directly). `#page_populate_with` would just call the accessor methods. This would then keep the behaviours consistent.